rocky/rocky/templates/oois/ooi_detail_origins_observations.html

```diff
@@ -20,6 +21,8 @@ <h2>{% translate "Observed by" %}</h2>
                             <td>
                                 {% if observation.boefje.id %}
                                     <a href="{% url 'boefje_detail' organization_code=organization.code plugin_id=observation.boefje.id %}">{{ observation.boefje.name }}</a>
+                                {% else %}
+                                    {% translate "The boefje has since been deleted / disabled." %}
```

Comment:
Just to be consistent with the rest of the strings
```suggestion
                                    {% translate "The boefje has since been deleted or disabled." %}
```